components/base/kinematicbase/differentialDrive.go

```diff
@@ -17,6 +17,7 @@ import (
 
 	"go.viam.com/rdk/components/base"
 	"go.viam.com/rdk/logging"
+	"go.viam.com/rdk/motionplan"
```

Comment:
I dont understand this question.  Why would adding this dependency be a problem?

Reply:
Sorry, I meant the entire `differentialDrive.go` file. The file changed but we don't have any way to exercise the code outside of tests is that right? So how do we know if it still works after this change?

Reply:
Good question. I'm going by inspection here.  The changes in this file should be purely a refactor (I hope that comes across), so whatever was happening should continue as it was.  

The only thing that would interfere with this would be if the underlying structures in the Plan changed.  It might be worth it to set up a hardware test of this to make sure it works end to end still